tests/benchdnn/graph/input_displacer.cpp

```diff
@@ -95,8 +147,26 @@ partition_data_displacer_t::partition_data_displacer_t(
                     break;
                 }
 
-                if (parent_op->kind_ == "Dequantize"
-                        || parent_op->kind_ == "DynamicDequantize") {
+                if (parent_op->kind_ == "DynamicDequantize"
+                        && dg.get_recognized_pattern()
+                                == graph_recognized_pattern_t::sdpa) {
+                    std::cout << "Detect Compressed SDPA\n";
```

Comment:
BENCHDNN_PRINT, please, if this message is important for the first level debugging.